app/models/manageiq/providers/redfish/physical_infra_manager/event_catcher.rb

```diff
@@ -0,0 +1,13 @@
+module ManageIQ::Providers::Redfish
+  class PhysicalInfraManager::EventCatcher < ::MiqEventCatcher
+    require_nested :Runner
+
+    def self.ems_class
+      PhysicalInfraManager
```

Comment:
I don't think this should be needed unless there's some reason the `ManageIQ::Providers::BaseManager::EventCatcher.ems_class` is wrong for some reason.

Reply:
:+1: that's what I was hoping